sumologic/resource_sumologic_monitors_library_monitor_test.go

```diff
@@ -106,6 +106,43 @@ func TestAccSumologicMonitorsLibraryMonitor_schemaValidations(t *testing.T) {
 	})
 }
 
+func TestAccSumologicMonitorsLibraryMonitor_timeFieldsSchemaValidation(t *testing.T) {
```

Comment:
@sumovishal Should I make acc test for all the fields I added validation for? It would be nice to have, but in my understanding (very basic) acc tests are costly.

Reply:
Yeah, I would not recommend an acceptance test for the validation. If you want to, I would suggest adding a unit test.

Reply:
Ok, I will remove it.